omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -273,9 +274,10 @@ protected PageableResult doSearch(RequestContext context) throws ResponseExcepti
 			else {
 				return new NeedsPaging<Order>(orders, context);
 			}
+		} else {
+			throw new InvalidSearchException("Please specify patient parameter with valid patientUuid");
```

Comment:
The error message reads as if one specified a patient parameter but with an invalid uuid.